packages/editor/src/components/post-title/index.native.js

```diff
@@ -43,6 +43,20 @@ class PostTitle extends Component {
 		this.props.onSelect();
 	}
 
+	onPaste( { value, onChange, html, plainText } ) {
+		const content = pasteHandler( {
+			HTML: html,
```

Comment:
Should we remove this line, or set html to undefined or null? When I tried pasting text with links, the links were showing up as blue in the title :thinking: WDYT?

Reply:
Good catch, going to remove it!